### ISSUE TITLE
unable to bookmark unnamed files

### ISSUE THREAD

**Issue by justrajdeep** [MattesGroeger/vim-bookmarks#102]
Hi

I usually redirect my `STDOUT` to vim using 

    $cmd | gvim -

But in the opened file the plugin does not work :(

since it overrides `m` key i am not able to put vim marks also.

**Comment by MattesGroeger** [comment]
Is this a particular issue with `gvim`? We don't really support this client.

**Comment by justrajdeep** [comment]
No, not exclusive to `gvim`. The issue is same with `vim` as well.

**Comment by MattesGroeger** [comment]
You are right, just tested myself. Will mark it as bug. Not sure why that happens. `ma` to open the list view works.
